crates/polars-core/src/frame/group_by/aggregations/mod.rs

```diff
@@ -480,20 +480,13 @@ where
                     } else if idx.len() == 1 {
                         arr.get(first as usize)
                     } else if no_nulls {
-                        Some(take_agg_no_null_primitive_iter_unchecked(
+                        take_agg_no_null_primitive_iter_unchecked::<_, T::Native, _, _>(
```

Comment:
Is it necessary to turbofish this call? If yes just disregard my comment.

Reply:
I think it's necessary to infer `TOut` since the closure isn't typed.